src/test/rgw/test_rgw_dmclock_scheduler.cc

```diff
@@ -424,6 +424,8 @@ TEST(Queue, SpawnAsyncRequest)
     EXPECT_EQ(PhaseType::priority, p2);
   });
 
+  context.poll();
+  // may require two poll invocations due to racy mutex locking
```

Comment:
This shouldn't be part of a librbd backport PR.

If this is an upstream change (I don't see it there), it should be backported separately.  If this is an octopus-specific fixup intended to make this test pass in the face of wider locking issues, even more reason to raise a separate rgw PR.

Reply:
@idryomov this is indeed an octopus-specific fixup to make this test pass with the backported change-set. I did add a comment about that in the commit message. I'm happy to create a separate PR if that is the preferred approach, but it seems a bit strange to me to create a separate pull request to fix tests that are failing due to the changes in this PR. Would the test fix PR be merged first and then the backport PR?

Reply:
You did mention it in the commit message but you didn't say where it came from.  I don't see a corresponding test fixup in master -- which commit was it cherry picked from?

If the test fixup is specific to octopus, what change(s) are missing in octopus that lead to this test failing?

Reply:
@cbodley Can you a take a quick look?

Reply:
Nevermind, looks like Cory hit https://tracker.ceph.com/issues/42788.